Dockerfile

```diff
@@ -1,5 +1,5 @@
-FROM grafana/grafana:2.6.0
-MAINTAINER Jimmi Dyson <jimmidyson@gmail.com>
+FROM grafana/grafana:latest
```

Comment:
@jimmidyson - we thought to rebase it with latest version always, or you want to nail it down to 4.0.2 ?